boards/stm32f429i-disc1/include/board.h

```diff
@@ -57,6 +57,13 @@ extern "C" {
 #define L3GXXXX_INT2_PIN    GPIO_PIN(PORT_A, 2) /**< INT2/DRDY pin used for L3Gxxxx */
 /** @} */
 
+/**
+ * @name stmpe811 driver parameters
```

Comment:
```suggestion
 * @brief stmpe811 driver parameters
```
I think this should be enough to make doxygen stop complaining